### DISCUSSION TITLE
Wrong MRT result

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#1100] [Questions & Problems]
After i get the results of my Rivaroxaban simulation , time values such as MRT and Thalf are highly overestimated but values like Cl are correct and as expected .one other problem is that MRT value in a single dose is so close to MRT value after administering rivaroxaban for 4 weeks.
i think kel is way underestimated how can i solve this problem ? 
Sincerely

**Comment by msevestre** [comment]
Hi @ranamjz 

You need to make sure your simulation time is long enough. We calculate the MRT using the last 10% point in your simulation. If the distribution phase is not finished, it will lead to wrong results

See https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-analysis-view

So in short, make sure you simulate long enough after your LAST infusion and your MRT values should be better

**Reply by ranamjz** [reply]
@msevestre thank you very much for this explanation but i ran the simulation for 4 weeks with the dosing interval of 24hrs meaning each patient is give 15mg of rivaroxaban orally every day for 4 weeks so i think it should be long enough but MRT is still highly overestimated  . Another thing is that this medication is administered orally and not intravenously .

**Reply by prvmalik** [reply]
@ranamjz is your reference calculating MRT with AUCinf or AUCtau? Make sure you are comparing apples to apples. PK-Sim is not likely set up to switch to AUCtau for multiple dose/steady state calculations of MRT.

**Reply by prvmalik** [reply]
Also check which profile PKSim is calculating it on if simulating for 4 weeks. Might not be the 28th dose, it may be more likely the first dose. Export the time profile to CSV and perform a manual calculation with some different methods to see which one it is.

**Reply by msevestre** [reply]
@ranamjz Let me try to rephrase. The time **AFTER** the last dose needs to be long enough. So it does not matter that you are dosing for 4 weeks or 4 days. If the simulation time ends right after the last dose, or too close to the last dose, your MRT calculation will be off. 
Please copy paste the graph so that we can eliminate this problem (please do you right click on the graph , copy instead of taking a picture of the application)

**Reply by ranamjz** [reply]
Dear @msevestre  , as you know when i want to introduce an advance protocol of administration to pk sim , i can choose the start time and number of répétitions plus the time between them and with these inputs pksim calculates the End time automatically so i cannot tell it to end my simulation with a gap from the last dosage

**Reply by ranamjz** [reply]
@msevestre and as for the graph i will copy it fully here ![thumbnail-1.png](https://user-images.githubusercontent.com/90548695/158404775-b846e6ed-8759-49b7-ad25-077d817cb6ba.png)

**Reply by ranamjz** [reply]
![20220315_183041.jpg](https://user-images.githubusercontent.com/90548695/158407145-7e42fca8-fe22-4019-8bed-739ca381758b.jpg)
As its evident here , i cannot change the end time

**Reply by msevestre** [reply]
End time in your simulation to simulate longer.  What you are showing in the `end time` of your application. Not the same

![image](https://user-images.githubusercontent.com/1041237/158408087-29fac6f0-6367-4e7d-8ec6-e843e62e8ed0.png)

**Reply by ranamjz** [reply]
@msevestre thank you very much i administered the medication for 2 weeks and i extended the last time to 2.5 weeks but i still see the errors .
![20220315_184736.jpg](https://user-images.githubusercontent.com/90548695/158410582-dcce54e1-923d-44a6-885d-8b5bbce425ff.jpg)

![thumbnail-2.png](https://user-images.githubusercontent.com/90548695/158410811-323d50bc-1482-4341-9cff-a9c05d745f2a.png)

**Reply by ranamjz** [reply]
![20220315_184801.jpg](https://user-images.githubusercontent.com/90548695/158410912-9652195a-5334-44b8-976b-f9a4fc8c066f.jpg)
![20220315_184947.jpg](https://user-images.githubusercontent.com/90548695/158411047-5e05cafc-2b6a-4a41-a4a0-278004cd6afc.jpg)

**Reply by msevestre** [reply]
This is not an error. Just a warning
"Absorption seems to be incomplete or absorption process is not finished. Vd, Vss (or Vd/F and Vss/F), t1/2, MRT and AUC_inf should be compared with respective PK-parameters from an IV simulation."

Your fraction absorbed is 0.21 that's why you are seeing this warning.

**Reply by ranamjz** [reply]
@msevestre Should i change my compound values to increase it? All of the values in ADME and other characteristics of my compound are from the literature but i get this massage how can i increase the fraction absorbed and achieve a correct MRT? because the values such as mrt , half life and understandably V values are far from correct
